cs130_front/src/components/views/ViewGroupPage/styles.css

```diff
@@ -0,0 +1,108 @@
+.panel {
```

Comment:
We don't need this file at all since you aren't changing any of the styles. The className= will search thru all CSS files. You can remove the import App.css from ViewGroupPage.js too.